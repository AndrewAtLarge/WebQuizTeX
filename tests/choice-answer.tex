% should give error as \answer inside choice environment
\documentclass{webquiz}
\title{A multiple choice question with a unique answer}
\begin{document}
  \begin{question}     % a quiz question
     What is the cube root of $64$?
     \begin{choice}[columns=3]% unique answer rendered in three columns
       \incorrect 1 \response No,  $1^3=1$
       \incorrect 2 \response No,  $2^3=8$
       \incorrect 3 \response No,  $3^3=27$
       \correct   4 \response Yes, $4^3=64$
       \incorrect 5 \response No,  $5^3=125$
       \incorrect 6 \response No,  $6^3=216$
       \incorrect 7 \response No,  $7^3=343$
       \incorrect 8 \response No,  $8^3=512$
       \incorrect 9 \response No,  $9^3=729$
       \answer Should not be here
     \end{choice}
  \end{question}
\end{document}
