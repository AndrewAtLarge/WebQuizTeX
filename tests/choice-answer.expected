! Package webquiz Error: You cannot have \answer inside a choice environment.
